src/search.js

```diff
@@ -1,5 +1,11 @@
 import Pipeline from './pipeline';
 
+// vanilla utilities
+const keys = Object.keys;
```

Comment:
I'm against aliasing these sorts of things -- it saves typing at the expense of clarity in the code.